mobile/Screen/EventScreen.js

```diff
@@ -25,29 +25,38 @@ const events = [{
   id: 1,
   title: 'Volunteering',
   date: '10/03/2023',
-  location: '3148 Rose Rd,LA',
+  day: 'Monday',
+  location: '3148 Rose Rd, LA',
   nonprofits: 'SRAA',
   description: 'helping artist hang artwork',
+  time: '1:00-2:00pm',
+  summary: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod temporincididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam aquis nostrud exercitation ullamcolaboris nisi ut aliquip ex ea commodo consequat.',
 },
 {
   id: 2,
   title: 'Fundraiser',
   date: '09/08/2023',
+  day: 'Tuesday',
   location: '4102 Daisy Rd, LA',
   nonprofits: 'SRAA, Apple',
   description: 'raising money for skid row artists',
+  time: '1:00-2:00pm',
+  summary: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod temporincididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam aquis nostrud exercitation ullamcolaboris nisi ut aliquip ex ea commodo consequat.',
 },
 {
   id: 3,
   title: 'Information Session',
   date: '03/05/2023',
+  day: 'Wednesday',
   location: '4123 Blue Rd, LA',
   nonprofits: 'SRAA, Google',
   description: 'information on the skid row artists',
+  time: '1:00-2:00pm',
+  summary: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod temporincididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam aquis nostrud exercitation ullamcolaboris nisi ut aliquip ex ea commodo consequat.',
 },
 ];
 
-function MapScreen({ navigation }) {
+function EventScreen({ navigation }) {
   const test = async () => {
```

Comment:
This function should be removed since it is never used and the example is not needed